src/shared/is_node.js

```diff
@@ -15,5 +15,7 @@
 /* globals module, process */
 
 module.exports = function isNodeJS() {
-  return typeof process === 'object' && process + '' === '[object process]';
+  return typeof process === 'object' &&
+    process + '' === '[object process]' &&
+    !process.versions['nw'];
```

Comment:
Can you change it to something like this for readability and explanation of why we exclude NW.js?

```javascript
// Exclude NW.js since, even though it's based on Node.js,
// it works like a browser.
return typeof process === 'object' &&
       process + '' === '[object process]' &&
       !process.versions['nw'];
```